service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseSubmissionControllerAboutToSubmitTest.java

```diff
@@ -43,6 +43,41 @@
     @Autowired
     private MockMvc mockMvc;
 
+    @Test
+    void shouldReturnErrorsWhenNoCaseDataIsProvided() throws Exception {
+        byte[] pdf = {1, 2, 3, 4, 5};
+        Document document = document();
+
+        given(userDetailsService.getUserName(AUTH_TOKEN))
+            .willReturn("Emma Taylor");
+        given(documentGeneratorService.generateSubmittedFormPDF(any(), any()))
+            .willReturn(pdf);
+        given(uploadDocumentService.uploadPDF(USER_ID, AUTH_TOKEN, pdf, "2313.pdf"))
+            .willReturn(document);
+
+        AboutToStartOrSubmitCallbackResponse response = makeSubmitCaseRequest("fixtures/emptyCaseData.json");
+
+        assertThat(response.getErrors()).isNotNull();
```

Comment:
But then we are testing the same on both int tests and unit tests? I thought this would be acceptable since int tests are more expensive. What do you think?

Reply:
I think you should at least check something - isNotNull is a bit confusing as this field is always notNull so no matter what your code does the test would pass. So I would probably include minimal tests, but yeah no reason to check much more.